src/scripts/services/auth.js

```diff
@@ -22,12 +25,11 @@ export default class Auth {
       if (authResult && authResult.accessToken && authResult.idToken) {
         this.setSession(authResult);
         window.location.reload();
-      }
-      if (err) {
+      } else if (err) {
+        this.logout();
```

Comment:
@nakedsushi about passing the custom error about user trying to access different domains.
I'm trying to do this https://github.com/auth0/auth0.js/issues/841

But so far no lucky with it.